java/src/test/java/Edition050_Faster_Android_Capabilities.java

```diff
@@ -0,0 +1,57 @@
+import org.junit.Test;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.io.IOException;
+import java.net.URL;
+
+import io.appium.java_client.AppiumDriver;
+
+public class Edition050_Faster_Android_Capabilities {
+
+    private String APP = "https://github.com/cloudgrey-io/the-app/releases/download/v1.8.0/TheApp-v1.8.0.apk";
+
+    private AppiumDriver driver;
+    private WebDriverWait wait;
+
+    @Test
+    public void testInstallingDependencies() throws IOException {
+        DesiredCapabilities caps = new DesiredCapabilities();
+
+        caps.setCapability("platformName", "Android");
+        caps.setCapability("deviceName", "Android Emulator");
+        caps.setCapability("automationName", "UiAutomator2");
+        caps.setCapability("app", APP);
+
+        driver = new AppiumDriver(new URL("http://localhost:4723/wd/hub"), caps);
+        wait = new WebDriverWait(driver, 10);
+
+        // Test code goes here
+
+        driver.quit();
+    }
+
+    @Test
+    public void testSkippingInstallOfDependencies() throws IOException {
+        DesiredCapabilities caps = new DesiredCapabilities();
+
+        caps.setCapability("platformName", "Android");
+        caps.setCapability("deviceName", "Android Emulator");
+        caps.setCapability("automationName", "UiAutomator2");
+
+        // App is already installed on device, so can be launched by Package name and Activity
```

Comment:
@Jonahss one curious question. In the 2nd test(testSkippingInstallOfDependencies) you mentioned that the app is already available. Do you mean that the app is loaded by the first test(testInstallingDependencies).
If that is the case, are we not assuming the sequence of execution of these tests?

Reply:
We are totally assuming that the tests will be run in sequence and not in parallel. Is there a way I could make that more clear?

Should I combine them into a single test case? Or make one depend upon the other explicitly?

Reply:
Thanks for clarifying that. 
I feel like may be it is a good idea if we can make one dependent on another, that way we can grantee the execution sequence.

Reply:
Good idea, thank you for the suggestion.

Reply:
@vsadineni1 How would you suggest I make them dependent. I could put the first test as an @Before, but then it's not clear that it's a test itself.

Reply:
@Jonahss the below link shows an example using the annotation: @FixMethodOrder(MethodSorters.NAME_ASCENDING)
https://github.com/junit-team/junit4/wiki/test-execution-order

Reply:
Oh that's perfect. Thanks.